build.sbt

```diff
@@ -7,15 +7,16 @@ name := "work-item-repo"
 
 makePublicallyAvailableOnBintray := true
 
-majorVersion                     := 6
+majorVersion                     := 7
 
 defaultSettings()
 
 scalaVersion := "2.11.12"
 
-crossScalaVersions  := Seq("2.11.12", "2.12.6")
+crossScalaVersions  := Seq("2.11.12", "2.12.10")
 
 libraryDependencies ++= LibDependencies()
+dependencyOverrides ++= LibDependencies.overrides.toSet
```

Comment:
the use of `apply` and `overrides` looks confusing - worth replacing apply with `compile ++ test`? or just rename?